README.md

```diff
@@ -1,7 +1,28 @@
 [![Coverity Scan Build Status](https://scan.coverity.com/projects/5120/badge.svg)](https://scan.coverity.com/projects/5120)
 
-# Tvheadend HTSP PVR
-Tvheadend HTSP PVR client addon for [Kodi] (http://kodi.tv)
+# Tvheadend PVR addon for Kodi
+
+This is a [Kodi] (http://kodi.tv) PVR addon for connecting to a (tvheadend)[https://tvheadend.org] backend.
+
+## Build instructions
+
+### Linux
+
+The following instructions assume you check out all repositories in a directory named `Projects` in your home 
+directory. cmake doesn't recognize `~` which is why `$HOME` is used when referring to your home directory.
+
+In addition to standard build tools you need to have the following packages installed: `kodi-addon-dev kodi-pvr-dev`. 
+These packages are available from Kodi's PPA repositories.
+
+1. `cd ~/Projects`
+2. `git checkout https://github.com/xbmc/xbmc.git`
+3. `git checkout https://github.com/kodi-pvr/pvr.hts.git`
+4. `cd pvr.hts && mkdir build && cd build`
+5. `cmake -DADDONS_TO_BUILD=pvr.hts -DADDON_SRC_PREFIX=/home/negge/Projects -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=/home/negge/Projects/xbmc/addons -DPACKAGE_ZIP=1 /home/negge/Projects/xbmc/project/cmake/addons/`
```

Comment:
cmake -DADDONS_TO_BUILD=pvr.hts -DADDON_SRC_PREFIX=../.. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../../xbmc/addons -DPACKAGE_ZIP=ON ../../xbmc/project/cmake/addons

No need to refer to abs paths that won't be valid.